src/dune_rules/warning.ml

```diff
@@ -95,3 +95,10 @@ let escaping_paths_in_install_stanza =
 ;;
 
 let no_keep_locs = create ~default:(fun _ -> `Enabled) ~name:"no_keep_locs" ~since:(3, 11)
+
+let deprecated_coq_lang_lt_08 =
```

Comment:
Actually, one small request: could you move this to somewhere in `dune_rules/coq`. I'd like to keep all the extensions separate

Reply:
I can move it, but note that I will have to export `Warning.create` too.

Reply:
I think that's fine. We need a public API for defining warnings anyway.